home-application/src/main/java/com/softserveinc/ita/homeproject/application/exception/mapper/ExceptionMapperUtils.java

```diff
@@ -0,0 +1,51 @@
+package com.softserveinc.ita.homeproject.application.exception.mapper;
+
+import cz.jirutka.rsql.parser.RSQLParserException;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.dao.InvalidDataAccessApiUsageException;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+public final class ExceptionMapperUtils {
+
+    public static String BadRequestExceptionMessageParser(RSQLParserException exception) {
```

Comment:
Please pay attention to the method naming!

Reply:
done